### ISSUE TITLE
Encrypted rar5 format is not supported I guess?

### ISSUE THREAD

**Issue by storm999** [adoconnection/SevenZipExtractor#67]
Encrypted rar5 format is not supported I guess?
Is it possible to add support for 'encrypted rar5'
 or rar5 itself does not allow this ?

**Comment by adoconnection** [comment]
Yeah, maybe newer version of 7z dlls will do the trick? I don’t recall anyone had any success with encrypted archives
